otter/grade/containers.py

```diff
@@ -158,7 +158,12 @@ def grade_assignments(submission_path, image, no_kill=False, pdf_dir=None, pdfs=
         if network is not None and not network:
             args['networks'] = 'none'
 
-        container = docker.container.run(image, command=["/autograder/run_autograder"], volumes=volumes, detach=True, **args)
+        container = docker.container.create(image, command=["/autograder/run_autograder"], **args)
+
+        for local_path, container_path in volumes:
+            docker.container.copy(local_path, (container, container_path))
+
+        docker.container.start(container, **args)
```

Comment:
I think you can remove `**args` from here -- `start` only has 2 kwargs, neither of which are set here.